src/apps/loyalty/containers/inquiries/mutations/update_conversation.ts

```diff
@@ -0,0 +1,47 @@
+import * as Relay from "react-relay"
+
+export type BuyerOutcome =
+  "PURCHASED" |
+  "STILL_CONSIDERING" |
+  "HIGH_PRICE" |
+  "LOST_INTEREST" |
+  "WORK_UNAVAILABLE" |
+  "OTHER"
+
+interface MutationProps {
```

Comment:
Nit: Now that these mutations have moved out of the component file, these interfaces could just be called `Props` again.